.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,4 +29,3 @@ Closes #XXXXX
 -   [ ] The `Because` section summarizes the reason for this PR
 -   [ ] The `This PR` section has a bullet point list describing the changes in this PR
 -   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
--   [ ] If this PR includes changes that needs to be updated on the `solutions` branch, I have created another PR (and linked it to this PR).
```

Comment:
I wonder if we should change this to something like this, because the purpose of this is to remind people that they may have to make changes in two places.

If this PR includes changes to exercises, they are also changes in the `solutions` folder.